### ISSUE TITLE
Review of the lab2 solution - done by Francesca Zafonte

### ISSUE THREAD

**Issue by Zafonte** [Matteo-Pietro-Pillitteri/Computational-Intelligence#1]
Hi Matteo, I reviewed your code and I'm glad to share my feedback. 

_**README:**_
The README file is well written and very explanatory of the problem and how you solved it.

_**CODE:**_
- The code is well formatted and easily readable. The names of variables, functions and comments are descriptive.

- In general the problem was carried out very well and there are not many things to say.

- I like the way you vary sigma and I suggest you try doing exploration and exploitation in the following way: use small sigma values if the new solution is close to the old one in terms of goodness, vice versa use large sigma values.

To summarize, I think you did a great job.

**Comment by Matteo-Pietro-Pillitteri** [comment]
Thanks Francesca for reviewing my code!
